src/dash-table/syntax-tree/lexeme/relational.ts

```diff
@@ -57,7 +57,8 @@ export const contains: IUnboundedLexeme = R.merge({
         op.toString().indexOf(exp.toString()) !== -1
     ),
     subType: RelationalOperator.Contains,
-    regexp: /^(contains)/i
+    regexp: /^((contains)(?=\s|$))/i,
```

Comment:
I had some browser support concerns but turns out lookahead is widely supported and not an issue. Lookbehind is not.
https://caniuse.com/#feat=js-regexp-lookbehind